### PR TITLE
Update pyaedt_cheat_sheet.tex

### PR DESCRIPTION
**Pull request by pvinod13**
I have added the final version of the AEDT-API Cheat Sheet, please have a look.

### CODE REVIEW

File: cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex [hunk 1]
```diff
@@ -1,22 +1,121 @@
-\documentclass[9pt,landscape]{article}
-\usepackage{{./../static/style}}
```
Comment (Revathyvenugopal162) [inline cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex:2]: Most of the style packages and renew commands(almost all) has been added in another style file `\usepackage{{./../static/style}}` so tis command will replace all the lines from 1 - 22 & 30-107. also the images are added there.

File: cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex [hunk 2]
```diff
@@ -29,43 +128,158 @@
...
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Analysis Class}
+The solution setup (\texttt{mysetup}) in HFSS design is analyzed by calling the following python command 
+\begin{lstlisting}[language=Python]
+hfss.analyze_setup("mysetup")
+\end{lstlisting}
+%where, 'mysetup' is the solution setup
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Post-processing Class}
+Post-processing class has modules for creating and editing plots in AEDT. They are accessible through the post library
+\begin{lstlisting}[language=Python]
+plotf = hfss.post.create_fieldplot_volume(object_list, quantityname, setup_name, intrinsic_dict) # This call return a FieldPlot object
```
Comment (MaxJPRey) [inline cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex:257]:
```suggestion
plotf = hfss.post.create_fieldplot_volume(object_list, quantityname, setup_name, intrinsic_dict) # This call returns a FieldPlot object
```

File: cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex [hunk 3]
```diff
@@ -29,43 +128,158 @@
...
+my_data = hfss.post.get_solution_data(expression=trace_names) # This call return a SolutionData object
```
Comment (MaxJPRey) [inline cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex:258]:
```suggestion
my_data = hfss.post.get_solution_data(expression=trace_names) # This call returns a SolutionData object
```

File: cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex [hunk 4]
```diff
@@ -29,43 +128,158 @@
...
+standard_report = hfss.post.report_by_category.standard("db(S(1,1))") # This call returns a new standard report object
+standard_report.create() # This call reate a report
+sols = standard_report.get_solution_data()
```
Comment (MaxJPRey) [inline cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex:261]:
```suggestion
solution_data = standard_report.get_solution_data()
```

File: cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex [hunk 5]
```diff
@@ -29,43 +128,158 @@
...
+\begin{lstlisting}[language=Python]
+hfss.mesh.assign_initial_mesh_from_slider(level=6) # setting the slider level to 6
+# assign model resolutions
+hfss.mesh.assign_model_resolution(names=[object1.name, object2.name], defeature_length=None)
+# assigning the mesh length to the object1 faces
+hfss.mesh.assign_length_mesh(names=object1.faces, isinside=False, maxlength=1, maxel=2000)
+\end{lstlisting}
+
+\section{\includegraphics[height=\fontcharht\font`\S]{slash.png} Analysis Class}
+The solution setup (\texttt{mysetup}) in HFSS design is analyzed by calling the following python command
```
Comment (MaxJPRey) [inline cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex:249]:
```suggestion
The solution setup (\texttt{mysetup}) in HFSS design is analyzed by calling the following Python command 
```

**Review by MaxJPRey — APPROVED** [review]
This is great @pvinod13 . Thanks for adding it for PyAEDT.

**Comment by RobPasMue** [comment]
What's the status here? @Revathyvenugopal162 @MaxJPRey @pvinod13

**Comment by MaxJPRey** [comment]
@RobPasMue As soon as @Revathyvenugopal162 suggestion's are merged, it is ready for merging in my opinion.

**Comment by RobPasMue** [comment]
@pvinod13 @Revathyvenugopal162 following @MaxJPRey's comment - can we get this PR updated and merged after solving comments?

**Comment by Revathyvenugopal162** [comment]
@pvinod13 We had to close on PR and opened another PR with combined AEDT and EDB API Cheatsheet? Is this the over written cheatsheet? I can see all these changes on the AEDT API there. IF both are the same, please add Maxime's suggestion to updated AEDT cheatsheet.

**Comment by Revathyvenugopal162** [comment]
@pvinod13 confirming that all changes have been incorporated into the latest cheatsheet. Sorry for the confusion. Please close the PR.

**Pull request closed without merging**